From f3b29efc4d7bb346cdd3fd79e780f1022c8f46e8 Mon Sep 17 00:00:00 2001
From: Tiago Vignatti <vignatti@c3sl.ufpr.br>
Date: Wed, 25 Jun 2008 02:07:49 +0000
Subject: Remove NOFONTSERVERACCESS macro

dix: this is semantically different but it goes through the same path in
the code.
---
diff --git a/configure.ac b/configure.ac
index 201bfe4..ab6cb10 100644
--- a/configure.ac
+++ b/configure.ac
@@ -962,7 +962,6 @@ fi
 
 if test "x$BUILTIN_FONTS" = xyes; then
        AC_DEFINE(BUILTIN_FONTS, 1, [Use only built-in fonts])
-       AC_DEFINE(NOFONTSERVERACCESS, 1, [Avoid using a font server])
        FONTPATH="built-ins"
 fi
 
diff --git a/dix/dixfonts.c b/dix/dixfonts.c
index 83b264a..89f1528 100644
--- a/dix/dixfonts.c
+++ b/dix/dixfonts.c
@@ -1920,8 +1920,6 @@ InitFonts (void)
         BuiltinRegisterFpeFunctions();
 #else
 	FontFileRegisterFpeFunctions();
-#endif
-#ifndef NOFONTSERVERACCESS
 	fs_register_fpe_functions();
 #endif
     }
diff --git a/include/dix-config.h.in b/include/dix-config.h.in
index 0c515c5..ad74f77 100644
--- a/include/dix-config.h.in
+++ b/include/dix-config.h.in
@@ -480,9 +480,6 @@
 /* Use only built-in fonts */
 #undef BUILTIN_FONTS
 
-/* Avoid using font servers */
-#undef NOFONTSERVERACCESS
-
 /* Use an empty root cursor */
 #undef NULL_ROOT_CURSOR
 
--
cgit v0.8.2
